### PR TITLE
chore: add hot step runtime test

### PR DESCRIPTION
**Pull request by LingyuCoder**
## Checklist

- [ ] Tests updated (or not required).
- [ ] Documentation updated (or not required).

### CODE REVIEW

**Comment by LingyuCoder** [comment]
!test-ng

**Comment by LingyuCoder** [comment]
!test-ng

File: .github/workflows/test-ng.yml [hunk 1]
```diff
@@ -53,9 +53,13 @@ jobs:
         run: pnpm run build:js
 
       - name: Test diff
-        id: testng
+        id: testdiff
```
Comment (LingyuCoder) [inline .github/workflows/test-ng.yml:56]: !test-ng

Comment (LingyuCoder) [reply .github/workflows/test-ng.yml:56]: !test-ng

Comment (LingyuCoder) [reply .github/workflows/test-ng.yml:56]: !test-ng

Comment (LingyuCoder) [inline .github/workflows/test-ng.yml:56]: !test-ng

**Comment by LingyuCoder** [comment]
* **#6335** <a href="https://app.graphite.dev/github/pr/web-infra-dev/rspack/6335?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#6318** <a href="https://app.graphite.dev/github/pr/web-infra-dev/rspack/6318?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @LingyuCoder and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by hardfist — APPROVED** [review]

**Pull request merged**